test/new_relic/agent/custom_event_aggregator_test.rb

```diff
@@ -97,6 +97,26 @@ def test_truncates_attributes
       assert_equal(expected, actual)
     end
 
+    def test_does_not_truncates_llm_embedding_input_attribute
+      params = {'input' => 'a' * 5000}
+      expected = {'input' => 'a' * 5000}
+
+      @aggregator.record(:LlmEmbedding, params)
+      actual = @aggregator.harvest![1].first[1]
+
+      assert_equal(expected, actual)
+    end
+
+    def test_does_not_truncates_llm_chat_message_content_attribute
+      params = {'content' => 'a' * 5000}
+      expected = {'content' => 'a' * 5000}
+
+      @aggregator.record(:LlmChatCompletionMessage, params)
+      actual = @aggregator.harvest![1].first[1]
+
+      assert_equal(expected, actual)
```

Comment:
Is the inverse (attributes not on the "do not truncate" list) being tested elsewhere? We want to make sure that the new logic only causes certain attributes to have their values truncated, but that all other values still get truncated.

Reply:
Yes ◡̈ Tanna added this test earlier when she added the truncation. Just a little [above](https://github.com/newrelic/newrelic-ruby-agent/pull/2511/files#diff-521cca09234a65ab1c95a0146b5022f8b7e6188da82503d130a0d7151b6660ebR84-R96) these tests.